pycc/tests/test_035_eomccsd.py

```diff
@@ -0,0 +1,64 @@
+"""
+Test EOM-CCSD solver.
+"""
+
+# Import package, test suite, and other packages as needed
+import psi4
+import pycc
+import pytest
+from ..data.molecules import *
+import numpy as np
+import sys
+
+np.set_printoptions(precision=10, linewidth=300, threshold=sys.maxsize, suppress=True)
+
+# H2O/cc-pVDZ
+def test_eomccsd_h2o():
+    # Psi4 Setup
+    psi4.set_memory('2 GB')
+    psi4.core.set_output_file('output.dat', False)
+    psi4.set_options({'basis': 'cc-pVDZ',
+                      'scf_type': 'pk',
+                      'mp2_type': 'conv',
+                      'freeze_core': 'false',
```

Comment:
we should make it a habit to check frozen core as well

Reply:
Added a frozen-core test case.  I've also added an ethylene test case to test more than one FC orbital (and pubchem access).